Makefile

```diff
@@ -1,6 +1,6 @@
 GIR_REPO := https://github.com/gtk-rs/gir.git
 GIR_VERSION := 2d1ffab19eb5f9a2f0d7a294dbf07517dab4d989
-OSTREE_REPO := https://github.com/fkrull/ostree.git
+OSTREE_REPO := ../ostree
```

Comment:
Hmm, what's this about? Is this supposed to be `https://github.com/ostreedev/ostree` instead?

Reply:
It was from an aborted attempt to get this code to use local git (it's not actually used right now).  Will update this more later.